utils/iio_attr.c

```diff
@@ -172,8 +172,12 @@ static int dump_channel_attributes(const struct iio_device *dev,
 					iio_channel_get_id(ch),
 					type_name);
 		}
-		if (iio_channel_get_name(ch) && quiet == ATTR_VERBOSE)
-			printf("id '%s', ", iio_channel_get_name(ch));
+		if (quiet == ATTR_VERBOSE) {
+			if (iio_channel_get_name(ch))
+				printf("id '%s', ", iio_channel_get_name(ch));
+			if (iio_channel_get_label(ch))
+				printf("label '%s', ", iio_channel_get_label(ch));
+		}
```

Comment:
could have just left the existing code, and added:

```
if (iio_channel_get_label(ch)) && quiet == ATTR_VERBOSE)
        printf("label '%s', ", iio_channel_get_label(ch));
```
would be less changes/churn.

Reply:
I see you point. I updated this.